ozaria/site/templates/play/play-level-view.pug

```diff
@@ -42,22 +41,10 @@ include game-dev-goals
 
       #game-dev-track-view
 
-      #level-chat-view
-
-      #multiplayer-status-view
-
-      #duel-stats-view
-
       #playback-view
 
-      #thang-hud
-
-      #level-dialogue-view
-
       button.btn.btn-lg.btn-primary.banner.header-font#stop-cinematic-playback-button(data-i18n="[title]common.continue;common.continue")
 
-    #surface-context-menu-view
```

Comment:
These removals are very good and very interesting. Should we remove all of the associated component?

For example. In this instance should [this code path](https://github.com/codecombat/codecombat/blob/b164782606b40a5c116e452ea6a21e811281c394/ozaria/site/views/play/level/PlayLevelView.js#L142) be cleaned up?

Non blocking. I think this is an excellent refactoring and wondering if it's in scope to delete more. Or if that's something we should do incrementally.

Reply:
Yeah, I agree that there is a lot of associated code for these removed elements, and I have just removed them from the jade file for now. I think we can delete the related code incrementally as and when appropriate. A lot of legacy clean up will follow the level UX re-design (will note this down for later). Thanks for pointing this out! :)